VideoAnalytics/VideoAnalyzer/Program.cs

```diff
@@ -88,7 +88,7 @@ private static async Task RunAsync(ConfigWrapper config)
             string inputAssetName = $"input-{uniqueness}";
 
             // Create a video analyzer preset with video insights.
-            Preset preset = new VideoAnalyzerPreset(null, InsightsType.VideoInsightsOnly);
+            Preset preset = new VideoAnalyzerPreset(null, null, InsightsType.VideoInsightsOnly);
```

Comment:
@xpouyat @Juliako  - should we add a comment here why we are passing in the two Null values.

Reply:
It seems introduced by the last AMS SDK. There a new (optional) parameter called ExperimentalOptions

Reply:
BTW, there are now 3 modes : audio only, video only and all.

Reply:
I will provide the parameters name so we don't expose null.

Reply:
Oh yeah, dont provide any details on the experimental options right now